docs/getting-started/getting-started-with-docker.rst

```diff
@@ -165,7 +295,7 @@ on building your Docker image on Quay.io we recommend their
 Build a Docker Image
 ~~~~~~~~~~~~~~~~~~~~
 
-Now that you've created the ``Dockerfile``, the next step is to build the image. Install `Docker Engine <https://docs.docker.com/engine/install/ubuntu/>`__ or `Docker Desktop <https://docs.docker.com/desktop/linux/install/>`__. Once it is installed, you can use this command to build your Docker image:
+Now that you've created the ``Dockerfile``, the next step is to build the image. Install `Docker Engine <https://docs.docker.com/engine/install/ubuntu/>`__ or `Docker Desktop <https://docs.docker.com/desktop/linux/install/>`__ if you have not done so already. Once it is installed, you can use this command to build your Docker image:
```

Comment:
It was like this, but I suggested a more OS-neutral link above.

Reply:
There is no OS-neutral link to installing Docker Engine, but I did change the Docker Desktop link.